test/powershell/engine/Help/UpdatableHelpSystem.Tests.ps1

```diff
@@ -431,6 +437,15 @@ Describe "Validate 'Update-Help' shows 'HelpCultureNotSupported' when thrown" -T
     }
 }
 
+# Describe "Validate 'Update-Help' shows 'FailedToUpdateHelpWithLocaleNoUICulture' when no language in culture" -Tags @('Feature')  {
+
+#     It 'Shows error if culture language is not specified for the current system:' -Skip:(($IsLinux -eq $false) || ((Get-Culture).LCID -ne 127)) {
+#         {
+#             Update-Help
+#         } | Should -Throw -ErrorId "FailedToUpdateHelpWithLocaleNoUICulture,Microsoft.PowerShell.Commands.UpdateHelpCommand"
+#     }
+# }
+
```

Comment:
Why is this test commented out?

Reply:
@daxian-dbw : I am having issues with the tests.

When I run that test file in my system (Windows AND WSL ubuntu) it works properly.

I use this command: 
`Start-Pspester ./test/powershell/engine/Help/UpdatableHelpSystem.Tests.ps1`

However, it seems to fail in the pipeline. I am considering removing it altogether.

One reason it might be failing is if the test environment is 'forcing' a culture. My test code skips if it is not linux or the culture is not the one with issues (LCID = 127), however, if the test system is adding a culture parameter my test will fail. Still... it works in my computer. 

Any advise?